apps/throttles.py

```diff
@@ -0,0 +1,48 @@
+from rest_framework.throttling import SimpleRateThrottle
+from influxdb_metrics.loader import log_metric
+
+
+class OrgBurst(SimpleRateThrottle):
+    """
+    API rate limiter for req/sec
+    """
+    scope = 'org_burst'
+
+    def get_cache_key(self, request, view):
+        if request.user.is_authenticated:
+            if 'payload' in request.user.token:
+                if 'organization_id' in request.user.token.payload:
+                    ident = request.user.payload.organization_id
+
+                    log_metric('transmission_api_throttling.info',
```

Comment:
Is this the right info that's needed for the logs?

Reply:
Include the organization ID in the metric tag

Reply:
I also don't think we need to include 'code' in this case

Reply:
Done.